<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TicketAddDialog</class>
 <widget class="QDialog" name="TicketAddDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1037</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="brandLabel">
       <property name="text">
        <string>Brand:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="brandEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="bEmptyLabel">
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Cannot be empty!</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="modelLabel">
       <property name="text">
        <string>Model:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="modelEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="mEmptyLabel">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Cannot be empty!</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="1">
      <widget class="QLineEdit" name="regIDEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="rEmptyLabel">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Cannot be empty!</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="regIDLabel">
       <property name="text">
        <string>Registration ID:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="descriptionLabel">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Desription of the issue:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="description">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>60</height>
        </size>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="emloyeesLabel">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Select employee responsible for repair</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="employees">
       <property name="maximumSize">
        <size>
         <width>16777213</width>
         <height>60</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QLabel" name="timeSelectLabel">
       <property name="text">
        <string>Choose when the repair will take place:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="legendLabel">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>White - free time slots,  Gray - unavailable time slots, Yellow -  time slots selected for this repair</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QTableWidget" name="calendar">
       <property name="maximumSize">
        <size>
         <width>780</width>
         <height>300</height>
        </size>
       </property>
       <row>
        <property name="text">
         <string>8:00-9:00</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>9:00-10:00</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>10:00-11:00</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>11:00-12:00</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>12:00-13:00</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>13:00-14:00</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>14:00-15:00</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>15:00-16:00</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>16:00-17:00</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>Mon</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Tue</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Wed</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Thu</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Fri</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Sat</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Sun</string>
        </property>
       </column>
       <item row="0" column="0">
        <property name="flags">
         <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
        </property>
       </item>
       <item row="1" column="0">
        <property name="flags">
         <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
        </property>
       </item>
       <item row="2" column="0">
        <property name="flags">
         <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
        </property>
       </item>
       <item row="3" column="0"/>
       <item row="4" column="0"/>
       <item row="5" column="0"/>
       <item row="6" column="0"/>
       <item row="7" column="0"/>
       <item row="8" column="0"/>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="createTicketButton">
     <property name="text">
      <string>Create ticket</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
